tools/depends/target/darwin-embedded-entitlements/Makefile

```diff
@@ -1,5 +1,5 @@
 include ../../Makefile.include
-DEPS= ../../Makefile.include Makefile darwin_embedded_entitlements.xml
+DEPS = ../../Makefile.include Makefile darwin_embedded_entitlements.xml ../../download-files.include
```

Comment:
No need to add the include here, but doesn't really matter if you want to keep it